chapters/introduction.tex

```diff
@@ -2,32 +2,20 @@ \chapter{Introduction}\label{introduction1}
 
 \section{Overview of Modelica}\label{overview-of-modelica}
 
-Modelica is a language for modeling of physical systems, designed to
-support effective library development and model exchange. It is a modern
-language built on acausal modeling with mathematical equations and
-object-oriented constructs to facilitate reuse of modeling knowledge.
+Modelica is a language for modeling of cyber-physical systems, designed to support effective library development and model exchange.
```

Comment:
Regarding "model exchange": It seems that FMI/FMU has overtaken Modelica Language (in number of supporting tools) when it comes to model exchange. Maybe we could also clarify the distinction w.r.t. FMI, say text-based, white-box or platform-neutral.

Reply:
Maybe "equation-based"?

Reply:
The next sentence talks about modeling with equations, so I think we don't need to mention it here.

Reply:
The same could probably be said regarding _support effective library development_ in view of the _facilitate reuse of modeling knowledge_.  One alternative could thus be to focus on the modeling aspect alone, but then combine with the following sentence to improve the text flow:
```suggestion
Modelica is a language for modeling of cyber-physical systems, based on acausal modeling with mathematical equations and object-oriented constructs to facilitate reuse of modeling knowledge.
```
(Note the switch from _built on_ to _based on_ to reduce risk of association with _cyber-physical systems_ instead of _language_.)

Reply:
Looks good to me. I'll wait for other comments and the official commit before approving.

Reply:
OK.  I'll give it some more time for reactions before pushing the change.

Reply:
Done.

Any concerns regarding the now complete absence of any form of _model exchange_?